src/components/DesignTheme/View/Commons/GenericCard.jsx

```diff
@@ -0,0 +1,64 @@
+import { useDispatch, useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { getContent } from '@plone/volto/actions';
+import { flattenToAppURL } from '@plone/volto/helpers';
+import moment from 'moment';
+import { Image } from 'semantic-ui-react';
+
+/**
+ * NewsCard view component class.
+ * @function Location
+ * @params {object} location: object.
+ * @returns {string} Markup of the component.
+ */
+const NewsCard = ({ item, showimage, image_field, content }) => {
+  const key = 'generic_card_' + item['@id'];
+  const url = flattenToAppURL(item['@id']);
+  const locationContent = useSelector(state => state.content.subrequests);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getContent(url, null, key));
+    return () => {};
+  }, [dispatch, item, url, key]);
+  let item_fo = null;
+  if (key in locationContent) {
+    item_fo = locationContent[key].data;
+  }
+
+  return item_fo ? (
+    showimage && item_fo[image_field] ? (
+      <div className="newscard card-img card card-teaser shadow p-4 mt-3 rounded border">
```

Comment:
Anche qui forse con una ``Card`` dal kit risolve con un po' di risparmio di smaronamento di attributi

Reply:
@pnicolli Ho 4 moduli in cui fare la sostituzione. Occorre che li vediamo insieme anche in altro momento: se uso le card del kit mi viene spaziata male la roba.